### DISCUSSION TITLE
<Default Starter> In Layout.js, It displays not the year of buildTime, but the year when a user accesses.

### DISCUSSION THREAD

**Discussion by twtwtw-gj** [gatsbyjs/gatsby#32505] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

In the sentence "build with Gatsby", It displays {new Date()}.

https://github.com/gatsbyjs/gatsby/blob/2dbe6477d88bf6e0138f247dc9dda1bd67a2a613/starters/default/src/components/layout.js#L42

I want to fix it with "buildTime", GraphQL property.

### Reproduction Link

https://github.com/gatsbyjs/gatsby

### Steps to Reproduce

1. Access https://gatsbystarterdefaultsource.gatsbyjs.io/
2. See the bottom words of Copyright

### Expected Result

@{the year when it built} , Built with Gatsby

### Actual Result

@{the year when a user accesses} , Built with Gatsby

### Environment

Does not depend on env.

**Comment by LekoArts** [comment]
You can access `site { buildTime }` via GraphQL: https://www.lekoarts.de/gatsby/tips-and-tricks-for-gatsby#date-of-last-build
